community/contributing/nightly-builds/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Nightly Builds
+layout: single
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+Bugs affecting more than one service commonly surface only when the whole system is running. Our continuous integration system builds and runs a suite of integration tests against Spinnaker nightly on real cloud provider infrastructure to detect these bugs.
+
+> **Access the CI system here**: [https://builds.spinnaker.io](https://builds.spinnaker.io)
+>
+> Requires membership in the `build-cops` GitHub Team to access the system.
+
+## Build Cop
+
+The **build cop** responsibilities include:
+
+* Triage integration test failures on the `master` and 3 most recent release branches
+* Clean up orphaned resources across target cloud providers
+* Route [new GitHub issues](https://github.com/spinnaker/spinnaker/issues) to the appropriate SIG
+  (applying GitHub labels as appropriate)
```

Comment:
:tada: This looks great! Maybe we can be a little more specific here and highlight that there is a pre-existing label for each SIG, including a link to the list of SIGs, and highlighting that the `sig-none` label should be applied if no existing SIG is a good fit.

Reply:
Added link to the SIG list.